src/store.js

```diff
@@ -287,6 +307,10 @@ function EvolvStore(options) {
     activeEids = result.activeEids;
 
     const newActiveKeys = objects.flattenKeys(effectiveGenome);
+
+    previousKeys.clear();
```

Comment:
Why not store the original set before reevaluating and then diff that against the result?

Reply:
I'm not following entirely. Nevertheless, for the `getActiveKeys.bind(this, activeKeys, previousKeys))` line later on to work, `previousKeys` has to exist at the time of binding and cannot be reassigned.